settings.gradle.kts

```diff
@@ -15,7 +15,7 @@ if(org.gradle.util.VersionNumber.parse(gradle.gradleVersion).major < 6) {
 if(gradle.parent == null) {
   includeBuildWithName("core", "pie.core.root")
   includeBuildWithName("lang", "pie.lang.root")
-  includeBuildWithName("bench", "pie.bench")
+//  includeBuildWithName("bench", "pie.bench") // todo: re-enable before merge
```

Comment:
Although needed to make it compile standalone, please don't comment this out. This should work when built as part of `devenv`, which is needed because this is the `develop` branch.

Reply:
re-enabled in the latest commit